src/features/hotspots/setup/OnboardingErrorScreen.tsx

```diff
@@ -29,7 +29,7 @@ const OnboardingErrorScreen = () => {
 
         <Text variant="h1">{t('hotspot_setup.onboarding_error.title')}</Text>
         <Text variant="body1" marginVertical="l">
-          {t('hotspot_setup.onboarding_error.subtitle')}
+          {t(`hotspot_setup.onboarding_error.subtitle.${connectStatus}`)}
```

Comment:
is there a fallback message if there's no i18n string defined for the `HotspotConnectStatus`?

Reply:
There isn't, but there should be.